posts/napari-cosmx-basics/using-napari-cosmx.qmd

```diff
@@ -0,0 +1,524 @@
+---
+title: "`napari-cosmx` essentials"
+author:
+  - name: Evelyn Metzger
+    orcid: 0000-0002-4074-9003
+    affiliations: 
+      - ref: nstg
+      - ref: eveilyeverafter
+toc: true
+toc-title: Contents
+toc-depth: 3
+toc-expand: 2
+toc-location: left
+number-sections: true
+number-depth: 4
+date: "2024-06-11"
+categories: [napari, how-tos, visualization]
+draft: false
+image: figures/fig-duality.png
+description: In this post, I walk through some of the basic 
+  ways I use napari-cosmx to view and analyze SMI data. I will make use of this
+  GUI/scripting duality and share a combination of GUI and programmatic tips and tricks.
+code-fold: false
+page-layout: full
+format: 
+  html:
+    theme: 
+      light: custom.scss
+      dark: darkly
+  pdf:
+    number-sections: true
+  docx:
+    toc: true
+    number-sections: true
+    highlight-style: github
+format-links: [pdf, docx]
+---
+
+```{r}
+#| eval: true
+#| echo: false
+#| label: "fig-duality"
+#| fig-cap: "Drawing that represents the duality of `napari-cosmx`. On the left side, cell types within a mouse coronal hemisphere are shown in an interactive Graphical User Interface. In addition to creating images interactively, the right side highlights that images can be generated programmatically. Both sides of `napari-cosmx` are discussed in this post."
+
+knitr::include_graphics("./figures/fig-duality.png")
+```
+
+<!-- napari-cosmx is flexible enough to quickly explore SMI data in a GUI -->
+<!--   yet robust enough to script reproducible results and tap into -->
+<!--   the underlying python objects.  -->
+
+# Introduction 
+
+This post is part of our napari series. In the [first blog post](../using-napari-for-cosmx-data/index.qmd){target="_blank"} I introduced the
+`napari-cosmx` plugin, how CosMx&#8482; Spatial Molecular Imager (SMI) data
+can be viewed as layers within napari, and a method for processing, 
+or "stitching", raw data that are exported from AtoMx&#8482; Spatial Informatics
+Portal (SIP). 
+
+One of the things that I love about the `napari-cosmx` plugin 
+is its duality. It's flexible enough to quickly explore SMI data in a Graphical
+User Interface (GUI) yet robust enough to script reproducible results and tap into
+the underlying python objects. In this post, I'll walk through some of the basic 
+ways in which we can use `napari-cosmx`to view SMI data. I'll make use of this
+duality and share a combination of GUI and programmatic tips-n-tricks.
+
+- @sec-example-data describes the example data that is publicaly available
+- @sec-working-with-layers shows you how to interact with viewer
+- @sec-scripting shows how we script to things to fine-tune image appearances and more
```

Comment:
should this be "how we script things"? no "to"?

Reply:
This comment is outdated.